web/gatsby-browser.js

```diff
@@ -1,6 +1,6 @@
+import '@fontsource/montserrat';
```

Comment:
Do we need to add this as well? This should be covered in the `/400.css` package.